src/snagrecover/recoveries/rockchip.py

```diff
@@ -0,0 +1,54 @@
+import usb
+import logging
+logger = logging.getLogger("snagrecover")
+from snagrecover.firmware.firmware import run_firmware
+from snagrecover.utils import get_usb
+from snagrecover.utils import cli_error
+from snagrecover.config import recovery_config
+import time
+
+
+def main():
+	usb_addr = recovery_config["usb_path"]
+	dev = get_usb(usb_addr)
+
+	# Blob made with boot_merger
+	if "xpl" in recovery_config["firmware"]:
```

Comment:
The output from `boot_merger` is typically called `loader`-images, and could be a better name for this option.

`xpl` can be confused with U-Boot's use of `xPL`, i.e. all TPL/VPL/SPL phases prior to U-Boot proper.